src/api/messages/getTemporaryFileUrl.js

```diff
@@ -0,0 +1,11 @@
+/* @flow strict-local */
+import type { Auth, ApiResponseSuccess } from '../transportTypes';
+import { apiGet } from '../apiFetch';
+
+type ApiResponseTempFileUrl = {|
+  ...ApiResponseSuccess,
+  url: string,
+|};
+
+export default (auth: Auth, filePath: string): Promise<ApiResponseTempFileUrl> =>
```

Comment:
A link to the documentation would be helpful here. I actually haven't found it yet; is it live? I've seen it in a file in zulip/zulip.git somewhere.

It would be natural for readers to have the same question you had, over here:

@_**Divyanshu Agrawal|13327** [said](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/view.20uploaded.20files/near/869441):
> The endpoint `/user_uploads/{realm_id_str}/{filename}` requires a parameter `realm_id_str` (integer). What is this, and where can I access this in the mobile codebase?

The detail from Tim's message, [here](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/view.20uploaded.20files/near/869498), seems key:

> I believe you should be able to just modify the URL of the file to append `/api/v1` at the start of the path in order to access this endpoint.

Maybe summarize this finding in a jsdoc, with a link to the documentation (if it's live).

Reply:
Okay, expanded the JSDoc with available information.